src/graph/graph.c

```diff
@@ -902,6 +902,114 @@ void Graph_GetNodeEdges
 	}
 }
 
+// returns node incoming/outgoing degree
+uint64_t Graph_GetNodeDegree
+(
+	const Graph *g,      // graph to inquery
+	const Node *n,       // node to get degree of
+	GRAPH_EDGE_DIR dir,  // incoming/outgoing/both
+	int edgeType         // relation type
+) {
+	ASSERT(g != NULL);
+	ASSERT(n != NULL);
+
+	GrB_Info info;
```

Comment:
I can't understand the point of having the "info" variable defined if it is unused.